orchestra/utils/task_lifecycle.py

```diff
@@ -1137,6 +1138,10 @@ def create_subsequent_tasks(project):
                             status=Task.Status.AWAITING_PROCESSING)
                 task.save()
 
+                # Apply todolist templates to Task
+                for template in task.step.todolist_templates_to_apply.all():
```

Comment:
add a small test to verify this runs:
* You can model the test off of  the first 3 lines of orchestra/utils/tests/test_task_lifecycle.py:BasicTaskLifeCycleTestCase.test_preassign_workers. Create a test_todolist_templates_to_apply that starts with those lines.
* You can create an empty TodoListTemplateFactory and link the step that will be created when you call `create_subsequent_tasks(project)` to this template.
* `patch` task_lifecycle.add_todolist_template and assert that it was called once with the slug of the `TodoListTemplateFactory`